assets/index.js

```diff
@@ -22,6 +22,9 @@ if ( !isTouchDevice() ) {
 var filterOpen = false;
 var previewOpen = false;
 
+window.manageTopics = document.createEvent('Event');
+window.manageTopics.initEvent('manage_topics', true, true);
```

Comment:
seems like it's [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Event/initEvent)